sql-plugin/src/main/scala/org/apache/spark/sql/rapids/execution/GpuHashJoin.scala

```diff
@@ -510,8 +510,7 @@ class ConditionalHashJoinIterator(
   }
 }
 
-
-class ExistenceJoinIterator(
+class MixedExistenceJoinIterator(
```

Comment:
This class is not only used with mixed joins, it's also used for any unconditional hash join.  I suggest calling this something like `HashedExistenceJoinIterator`

Reply:
done